F1WM/ApiModel/Engines/Engines.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using F1WM.DatabaseModel;
+
+namespace F1WM.ApiModel.Engines
+{
+	public class Engines
+	{
+		public IEnumerable<Engine> EnginesList { get; set; }
```

Comment:
This is a kind of a shortcut we'd rather want to avoid. E.g. there is no point in returning the letter in every Engine api model, engineMakeId also seems to be redundant. Please create an appropriate api model for Engine, instead of using database model here.